topic_imu_in: /livox/imu
topic_imu_out: /imu/data
lidar:
  topic_lidar_in: /livox/lidar
  topic_lidar_out: /livox/raw
  topic_lidar_time_align: /livox/align
g: 9.80511
lidar0:
  T_imu_lidar: [
    1, 0, 0, 0.011,
    0, 1, 0, 0.02329,
    0, 0, 1, -0.04412,
    0, 0, 0, 1]
  rostopic: /livox/lidar
max_cameras: 2
refer_cam_id: 0 # or 1
cam0:
  T_lidar_cam: [
    0.8475287815717099, -0.4456535402996677, 0.288249694580835, 0.02192650352544739,
    0.3246906087435981, 0.005738192152769911, -0.9458028767901527, -0.07112085923538626,
    0.4198463683316601, 0.895187128576584, 0.1495628022923002, -0.03290826958424917,
    0, 0, 0, 1]
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.009271881530398384, -0.002256102158021696, -4.368206575029892e-05, -0.0004267565532564276]
  distortion_model: equidistant
  intrinsics: [1191.142108872901, 1191.101711361493, 1536.632173853755, 1982.118393042963]
  in_resolution: [3040, 4032]
  rostopic: /camera/left/jpeg
  timeshift_cam_imu: 0.0
  rostopic_out: /camera/left/jpeg_1k/data
  rostopic_out_undistort: /camera/left/jpeg_1k/undistort
  out_resolution: [760, 1008]
cam1:
  T_lidar_cam: [
    -0.8542218288651381, -0.4310341968044736, -0.2907139285886052, -0.01917998503065129,
    0.3291803494297044, -0.01558775757347314, -0.944138400534142, -0.0647611036151011,
    0.4024243589044232, -0.9022009438029605, 0.1552033903025672, -0.03538904437538264,
    0, 0, 0, 1]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.01008649795949601, -0.001610579611010701, -0.0003634725781638666, 0.0003366160363341307]
  distortion_model: equidistant
  intrinsics: [1192.048568788325, 1191.749865739601, 1535.394598691759, 2067.27558021998]
  in_resolution: [3040, 4032]
  rostopic: /camera/right/jpeg
  rostopic_out: /camera/right/jpeg_1k/data
  rostopic_out_undistort: /camera/right/jpeg_1k/undistort
  out_resolution: [760, 1008]
  timeshift_cam_imu: 0.0
